claim/README.md

```diff
@@ -114,24 +115,32 @@ docker run -d --name=netdata \
   -v /proc:/host/proc:ro \
   -v /sys:/host/sys:ro \
   -v /etc/os-release:/host/etc/os-release:ro \
+  -e NETDATA_CLAIM_TOKEN=TOKEN \
+  -e NETDATA_CLAIM_URL=https://app.netdata.cloud \
```

Comment:
@joelhans in all cases the claim URL needs to be in double quotes such as "https://app.netdata.cloud" Cc @Ferroin

Reply:
This was actually my mistake.